src/System.Console/ref/System.Console.cs

```diff
@@ -36,8 +36,11 @@ public static partial class Console
         public static void MoveBufferArea(int sourceLeft, int sourceTop, int sourceWidth, int sourceHeight, int targetLeft, int targetTop, char sourceChar, ConsoleColor sourceForeColor, ConsoleColor sourceBackColor) { }
         public static bool NumberLock { get { return default(bool); }}
```

Comment:
Ok.  I thought maybe we were doing something across all libraries in the dev branch before merging it back to master.  Otherwise we're going to need to keep track of which contracts we've bumped as part of this effort?